apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -0,0 +1,963 @@
+    
+    useEffect(() => {
+        setSelectedUserStore(userstore);
+        getUserStoreList();
+    }, [ userstore ]);
+
+    /**
+     * Fetches SCIM dialects.
+     */
+    const getSCIMDialects = async (): Promise<ClaimDialect[]> => {
+        setIsLoading(true);
+
+        try {
+            // Get SCIM dialect URIs for the user resource.
+            const resourceResponse: any = await getSCIMResourceTypes();
+            const scimResources: SCIMResource[] = resourceResponse?.Resources;
+            let scimDialectsSchemas: string[];
+
+            for (const resource of scimResources) {
+                if (resource?.id === "User") {
+                    const schemaExtensions: SCIMSchemaExtension[] = resource?.schemaExtensions;
+
+                    scimDialectsSchemas = schemaExtensions.map((extension: SCIMSchemaExtension) => {
+                        return extension.schema;
+                    });
+
+                    scimDialectsSchemas.push(resource?.schema);
+                }
+            }
+
+            const dialectResponse: ClaimDialect[] = await getDialects({});
+
+            // Filter only the SCIM dialects.
+            const scimDialects: ClaimDialect[] = dialectResponse.filter((dialect: ClaimDialect) => {
+                return scimDialectsSchemas.includes(dialect.dialectURI);
+            });
+
+            return scimDialects;
+        } catch (error) {
+            setHasError(true);
+            dispatch(
+                addAlert({
+                    description:
+                        error?.response?.data?.description ||
+                        t(
+                            "console:manage.features.claims.dialects.notifications.fetchDialects" +
+                                ".genericError.description"
+                        ),
+                    level: AlertLevels.ERROR,
+                    message:
+                        error?.response?.data?.message ||
+                        t(
+                            "console:manage.features.claims.dialects.notifications.fetchDialects" +
+                                ".genericError.message"
+                        )
+                })
+            );
+        } finally {
+            setIsLoading(false);
+        }
+    };
+
+    /**
+     * This will fetch external claims for each dialect
+     * and create a list of already mapped local claims.
+     */
+    const getClaimMapping = async (): Promise<CSVAttributeMapping[]> => {
+        const scimDialects: ClaimDialect[] = await getSCIMDialects();
+
+        const scimClaimPromises: Promise<ExternalClaim[]>[] = [];
+
+        scimDialects.forEach((dialect: ClaimDialect) => {
+            scimClaimPromises.push(getAllExternalClaims(dialect.id, null));
+        });
+
+        try {
+            setIsLoading(true);
+            const scimClaimResponse: ExternalClaim[][] = await Axios.all(scimClaimPromises);
+            const _attributeMapping: CSVAttributeMapping[] = [];
```

Comment:
To make it consistent with other variable names
```suggestion
            const attributeMapping: CSVAttributeMapping[] = [];
```